iOS-Settings/Sources/App/Setting.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Setting.swift
+//  iOS-Settings
+//
+//  Created by Kuanysh al-Khattab Auyelgazy on 08.03.2023.
+//
+
+import UIKit
+
+enum SettingType {
+    case plain
+    case detailed
+    case switcher
+}
+
+struct Setting {
+    let type: SettingType
+    let name: String
+    let detail: String?
+    let iconImage: UIImage
```

Comment:
It is not recommended to use UI elements in your models. You can change it to let iconImage: String, then when you will setup view, you can use it like this
let image = UIImage(named: setting.iconImage)

For UIColor, you can create Enum of colors that you are going to use, then you can map your enum to UIColor when you will set up your view